### ISSUE TITLE
proto format and instructions seem out of date

### ISSUE THREAD

**Issue by davidalpert** [darcys22/godbledger#111]
using the latest version of the protobuf package on MacOS installed via `brew install protobuf`
```
$: protoc --version
libprotoc 3.14.0
```
when I follow the [`Building the Proto Buffers` instructions](https://github.com/darcys22/godbledger#building-the-proto-buffers) on the readme:
```
protoc -I proto/ proto/transaction.proto --go_out=plugins=grpc:proto
```
I get errors:
```
2020/11/28 06:27:20 WARNING: Missing 'go_package' option in "transaction.proto",
please specify it with the full Go package path as
a future release of protoc-gen-go will require this be specified.
See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.

--go_out: protoc-gen-go: plugins are not supported; use 'protoc --go-grpc_out=...' to generate gRPC
make: *** [proto] Error 1
```

it appears that:
- the syntax of the `protoc` command is out of date; and
- the format of the `transaction.proto` file is missing a declaration

probably the way that proto options are handled has changed in an update to the protobuf toolchain since these files were last generated.

**Comment by davidalpert** [comment]
following the instructions on the [Quick start - gRPC - Go](https://grpc.io/docs/languages/go/quickstart/#regenerate-grpc-code) page, I tried the following command:
```
protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative proto/transaction.proto
```
and adding the following to `./proto/transaction.proto`
```
option go_package = "github.com/darcys22/godbledger/transaction";
```
seemed to work.

pull request inbound.

**Comment by davidalpert** [comment]
this looks a bit more complicated than just updating the syntax of the `protoc` command, as it seems to require an update of:
- `github.com/golang/protobuf` from `v1.4.2` to `v1.4.3`
- `google.golang.org/grpc` from `v1.30.0` to `v1.33.2`
and the introduction of a `google.golang.org/protobuf` dependency at `v1.25.0`

TL;DR still testing locally

**Comment by davidalpert** [comment]
PR #112 looks good on my machine.

**Comment by darcys22** [comment]
Thank you!!!
